### ISSUE TITLE
WCAG 3.0 draft review: Response to Introduction question on what more to move to the explainer (if anything)

### ISSUE THREAD

**Issue by maryjom** [w3c/silver#600]
IBM Response to the first requested input posed in **Section 1. Introduction** of the 7 Dec. 2021 draft.

### Question 1
Is there more (or less) introductory content that should be moved to the Explainer document? 

**Answer:** I think that the Appendix section B Differences from WCAG 2 and its subsections B.1 and B.2 could be moved to the explainer. Note: if the language didn't change as completely, these explanations and mappings would not be necessary.

Additionally, the listing of functional categories per criteria could also be moved either to the explainer, to the document on functional categories, or into an appendix. The reason is that the short phrases to describe the functional category are sometimes not self-explanatory, and are repeated in the main WCAG 3.0 document as well as in the Outcomes page for each guideline. 
This could be implemented similar to how the \ EN 301 549 handled the Relationships between requirements and the functional performance statements in Annex B.  I think this will also be helpful to assessors trying to determine bronze/silver/gold conformance to more easily know which SC to  aggregate results from to check for conformity to a particular functional category. 

I have specific comments on the functional categories that are included in [Issue 601](https://github.com/w3c/wcag3/issues/250)

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).

**Comment by maryjom** [comment]
Agree, this can be closed.
